exegol/model/ContainerConfig.py

```diff
@@ -296,6 +309,35 @@ def interactiveConfig(self, container_name: str) -> List[str]:
 
         return command_options
 
+    def enableSound(self):
+        """Procedure to enable sound feature"""
+        if not SoundUtils.isPulseAudioAvailable():
+            logger.error("Sound sharing feature is [red]not available[/red] on your environment. [orange3]Skipping[/orange3].")
+            return
+        if not self.__enable_sound:
+            logger.verbose("Config: Enabling sound sharing")
+            try:
+                self.addVolume(SoundUtils.getPulseAudioSocketPath(), "/run/user/0/pulse/native", must_exist=False)
```

Comment:
Why must_exist is set to False here ? Shouldn't be True instead ? 
If the socket is not present, the feature cannot be activated.

Reply:
Just below, he left a commented FIXME, indicating that a solution must be found here. In it's current state and in this context, the `addVolume` function raises an error when `must_exist` is True 😉